packages/website/components/hero.js

```diff
@@ -7,17 +7,13 @@ import HeroIllustration from '../illustrations/hero-illustration'
 import HeroBackgroundLeft from '../illustrations/hero-background-left'
 import HeroBackgroundRight from '../illustrations/hero-background-right'
 
-const supports = typeof CSS !== 'undefined' && CSS.supports
-  ? CSS.supports
-  : () => true
-
 export default function Hero() {
   /** @type {import('react').CSSProperties} */
-  const style = supports('overflow: clip') ? { overflowX: 'clip' } : { overflow: 'hidden' }
+  const style = { overflowX: 'clip' , overflow: 'hidden' }
```

Comment:
I found that this hid the overflow in the y axis and so the title below the hero didn't have the usual background swirls and lines.

Reply:
UPS, I forgot to push `overflowY: 'visible' 😅